src/screens/admin/Post.js

```diff
@@ -1,32 +1,28 @@
-import React from 'react'
-import { Link, useParams, useHistory } from 'next/link'
+import React from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import usePost from '../../hooks/usePost';
+import useSavePost from '../../hooks/useSavePost';
+import useDeletePost from '../../hooks/useDeletePost';
+import PostForm from '../../components/post-form';
+import { Loader } from '../../components/styled';
 
-//
+export default function AdminView() {
+  const { query: { adminId }, push } = useRouter();
```

Comment:
Minor point however I would still think that this should be named postId as that is what it is referencing even if in the admin section. Changing the name suggests it's not a post by extension. Super minor though.